.coveragerc

```diff
@@ -1,7 +1,34 @@
 [run]
+parallel = true
 source = EXOSIMS
+data_file = coverages/.coverage
 omit = 
+  tests
+  */__init__.py
```

Comment:
I ignored */__init__.py files as most of them were empty, and made certain modules show up with zero coverage when there was really nothing to cover.